Cargo.lock

```diff
@@ -62,7 +62,7 @@ version = "0.1.10"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "9c041a8d9751a520ee19656232a18971f18946a7900f1520ee4400002244dd89"
 dependencies = [
- "critical-section",
+ "critical-section 0.2.7",
```

Comment:
Was this updated as part of a dependency chain for the new riscv?

Reply:
Yea. As part of the bump, a new version of `critical-sections` was pulled in, but something else was still depending on the old. (`riscv-rt` I think)

Reply:
Ack